README.rst

```diff
@@ -219,14 +219,14 @@ file for more info.
     :target: https://pypi.org/project/auth0-python/
     :alt: Latest version released on PyPI
 
-.. |coverage| image:: https://codecov.io/gh/auth0/auth0-python/badge.svg
-    :target: https://codecov.io/gh/auth0/auth0-python
-    :alt: Test coverage
-
-.. |build| image:: https://circleci.com/gh/auth0/auth0-python.svg?style=shield&circle-token=:circle-token
+.. |build| image:: https://img.shields.io/circleci/project/github/auth0/auth0-python.svg?style=flat-square&label=circleci
     :target: https://circleci.com/gh/auth0/auth0-python
     :alt: Build status
 
-.. |license| image:: http://img.shields.io/:license-mit-blue.svg?style=flat
+.. |coverage| image:: https://img.shields.io/codecov/c/github/auth0/auth0-python.svg?style=flat-square&label=codecov
+    :target: https://codecov.io/gh/auth0/auth0-python
+    :alt: Test coverage
+
+.. |license| image:: https://img.shields.io/:license-mit-blue.svg?style=flat-square
     :target: http://doge.mit-license.org
```

Comment:
Can you update this to https://opensource.org/licenses/MIT while you're in there?